evidence/skunert/0003-skunert-retain-II-2024.md

```diff
@@ -0,0 +1,77 @@
+# Evidence-0003: Retention at Rank II
+
+|                  |            |
+| ---------------- | ---------- |
+| **Report Date**  | 2024/10/28 |
+| **Submitted by** | skunert    |
+
+
+## Member details
+
+- Matrix username: `@sebastian:parity.io`
+- Polkadot address: `1682A5hxfiS1Kn1jrUnMYv14T9EuEnsgnBbujGfYbeEbSK3w`
+- Current rank: 2
+- Date of initial induction: 2024/01/08
+- Date of last report: 2024/07/29
+- Area(s) of Expertise/Interest: `Cumulus Node`, `Substrate Node`, `Parachain Authoring`, `Weight`
+
+
+## Reporting period
+
+- Start date: 2024/08/24
+- End date: 2024/10/24
```

Comment:
I know the recommendation is to put up this letter 30 days before retention deadline, but it kinda leads to having only two months of activity reporting..

I personally put it up a bit later (like 15-20 days before) and I include the previous three months from posting date, aka `[now-3months, now]` so that I can capture full 3 months of activity for each report

Reply:
Yeah, the report was actually already about the last 3 months. The begin date I took from my last report which was the mistake here.